diagnostic.md

```diff
@@ -5,43 +5,44 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+// It is to make a web app functional and interactive
```

Comment:
In a way yes, `One purpose of a backend is to allow users on the same web app to interact with each other.
Without a backend, users would not be able to do this, limiting what our application is capable of. Another
purpose is to securely store data that can persist over time.` So it does add significantly more functionality.